### PR TITLE
fixed some problems with warnigs, fixed minor typo(s)

### PR DESCRIPTION
**Pull request by HenryDane**
I went ahead and added some (hopefully) useful changes to how warnings are handled -- I am not sure if this is what you had in mind but it may be useful.

I also fixed a typo in the `m2ihm(self, minuite)` function.

### CODE REVIEW

File: pyhecdss/pyhecdss.py [hunk 1]
```diff
@@ -232,6 +232,21 @@ def _get_istat_for_zrrtsxd(self, istat):
...
+    def _respond_to_istat_state(self, istat):
+        if istat == 0:
+            # everything is ok
+            pass
+        elif istat == 1 or istat == 2 or istat == 3:
+            warnings.warn("Some data or data blocks are missing [istat=" + istat + "]", RuntimeWarning)
+        elif istat == 4:
+            warnings.warn("Found file but failed to load any data", RuntimeWarning)
+        elif istat == 5:
+            # should this be an exception?
```
Comment (dwr-psandhu) [inline pyhecdss/pyhecdss.py]: Agree, this should be  an exception.
`raise RuntimeError(msg)`

Comment (HenryDane) [reply pyhecdss/pyhecdss.py]: I've updated this and pushed the changes to this pull request.

**Comment by HenryDane** [comment]
I've updated the code to raise an exception and to import the `warnings` module.

File: pyhecdss/pyhecdss.py [hunk 2]
```diff
@@ -80,6 +81,7 @@ def get_version(self):
         """
         #needs to be done on a closed file
         if (self.isopen): self.close()
+        pyheclib.zfver_(self.fname, char *cver, int *iver, len(fname), len(cver));
```
Comment (dwr-psandhu) [inline pyhecdss/pyhecdss.py]: This line should be removed. Old code. The new one is merged below it.

**Comment by HenryDane** [comment]
Everything should be fixed now.

**Pull request merged**